public/client-v2/src/constants/priorities.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+import type { Priorities } from '../types/Fronts.js';
+
+export const priorities: Priorities = {
```

Comment:
Perhaps we could remove priorities from the front config entirely if we're specifying them by hand here? Seems like a duplication for no reason?

Reply:
The priorities here are settings for each priority we are displaying on the client. They are empty objects for now as we only need their names but they will have some data in them in the future. The priorities of fronts coming back in the fronts config are just names of priorities for each front and the priority types are just making sure that priorities are always conforming to the shape here. Maybe it makes sense to export the priorities type from this file rather than from `types/Fronts` ?